views/pages/toplevel/home.njk

```diff
@@ -25,75 +28,73 @@
 {% block content %}
 
     {% set heroContent %}
-        {{ headers.overlayText('h2', "The Big Lottery Fund is the largest funder of community activities in the UK. We distribute money raised by players of the <a href=\"https://www.national-lottery.co.uk\">National Lottery</a>", 'overlay-text--has-links t2') }}
+        <div class="o-button-group">
+            <a class="btn btn--small [ accent--white a--btn ]"
```

Comment:
This square bracket thing is a method for grouping based on https://csswizardry.com/2014/05/grouping-related-classes-in-your-markup/ 

I was finding the distinction if different _types_ of classes hard to read so was aiming to differentiate between the unique module classes and modifiers. Happy to remove this if you find it to be too noisy.